src/dvclive/live.py

```diff
@@ -95,8 +95,11 @@ def __init__(
         self._report_notebook = None
         self._init_report()
 
-        self._baseline_rev: Optional[str] = None
-        self._exp_name: Optional[str] = exp_name
+        self._baseline_rev: str = os.getenv(
+            env.DVC_EXP_BASELINE_REV,
+            "0" * 40,  # noqa: PLW1508
```

Comment:
[Q] Can you create a variable out of this so that we at least have some breadcrumbs for the knowledge? I think we should add a reference to what 40 0s is in Studio as well because this will either get buried or lost.

Reply:
could be `NO_REPO_SHA` or similar